src/accessor/mod.rs

```diff
@@ -109,8 +109,10 @@ impl<'a> Accessor<'a> {
     }
 
     /// Returns the buffer view this accessor reads from.
-    pub fn view(&self) -> buffer::View<'a> {
-        self.document.views().nth(self.json.buffer_view.value()).unwrap()
+    ///
+    /// This may be `None` if the corresponding accessor is sparse.
+    pub fn view(&self) -> Option<buffer::View<'a>> {
+        self.json.buffer_view.map(|view| self.document.views().nth(view.value()).unwrap())
```

Comment:
We need to add a validation check for this otherwise the unwrap might fail. A new `Validate` implementation will be required for `gltf_json::Accessor` to replace the [automatically derived one](https://github.com/gltf-rs/gltf/blob/36fbd771acad62d7f8a633fec32ebdefdf6f3d4a/gltf-json/src/accessor.rs#L191).

Reply:
That unwrap is for the nth call for indexing buffer views (I think). It was there before this PR. Do you mean that we need a validation for checking that buffer_view can only be omitted if sparse is present? Anyway I added a tentative custom Validate impl for your review.

Reply:
Yes, that's what I meant. This looks good, thanks.